packages/fx-core/tests/plugins/solution/solution.arm.test.ts

```diff
@@ -21,15 +21,24 @@ import {
   HostTypeOptionSPFx,
   TabOptionItem,
 } from "../../../src/plugins/solution/fx-solution/question";
-import { generateArmTemplate } from "../../../src/plugins/solution/fx-solution/arm";
+import {
+  deployArmTemplates,
+  generateArmTemplate,
+} from "../../../src/plugins/solution/fx-solution/arm";
 import { it } from "mocha";
 import path from "path";
 import { ArmResourcePlugin } from "../../../src/common/armInterface";
+import mockedEnv from "mocked-env";
+import { UserTokenCredentials } from "@azure/ms-rest-nodeauth";
+import { ResourceManagementModels, Deployments } from "@azure/arm-resources";
+import { HttpHeaders } from "@azure/storage-blob";
```

Comment:
iS HttpHeaders exists in other package? import from storage package seems strange

Reply:
Fixed.